### DISCUSSION TITLE
Issue creation links with template are not handled correctly

### DISCUSSION THREAD

**Discussion by Craftplacer** [community/community#36533] [Mobile]
I develop an app which allows users to report in-app errors straight to GitHub and I have noticed that the GH app intercepts them and doesn't handle them correctly as intended.

For example, my app would open `github.com/user/repo/issues/new?template=error-template.yml` but the GitHub app only displays a list of templates to choose from, the information encoded in the URL isn't handled at all.

I am currently on v1.84.1

**Comment by iskae** [comment]
Hi @Craftplacer 👋 

Thanks so much for reaching out ❤️ 

You are totally right 💯 We are not handling deeplinks for filling the templates out directly for now, but I believe that this is a great idea and would be a nice addition to our apps 😊 

I have created an internal issue for this and we will try to add it in our upcoming releases 👍 Please stay tuned 🙇 

Cheers!
